packages/react-components/src/components/AppFrame/types.ts

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+
+export interface INavBarListProps {
+  className?: string;
+}
+
+export interface INavBarOption {
```

Comment:
I'd suggest storing the types near their actual usage - in the `<Component>/types.ts` files